app/services/sign_in/session_creator.rb

```diff
@@ -103,10 +106,37 @@ def credential_email
       @credential_email ||= validated_credential.credential_email
     end
 
+    def user
+      @user ||= User.find(user_uuid)
```

Comment:
I don't think we should do a `User.find` during auth, it makes this dangerously close to a loop that could break all of this (in fact I kinda want to remove `User` creation from auth altogether. I don't think we need to do it until our first authenticated call)